src/pieces/commands/autocommit.py

```diff
@@ -0,0 +1,80 @@
+import subprocess
+import re
+from pieces.gui import show_error
+from pieces.api.config import pos_client,api_client
+import re
+
+def get_current_working_changes(word_limit:int=2000) -> str:
```

Comment:
would be awesome to get some comments on a couple things: the expected behavior, any sort of limitations, a general description of this command